packages/block-library/src/comments-pagination-next/index.php

```diff
@@ -26,7 +26,7 @@ function render_block_core_comments_pagination_next( $attributes, $content, $blo
 	$label            = isset( $attributes['label'] ) && ! empty( $attributes['label'] ) ? $attributes['label'] : $default_label;
 	$pagination_arrow = get_comments_pagination_arrow( $block, 'next' );
 
-	$filter_link_attributes = function() {
+	$filter_link_attributes = static function() {
 		return get_block_wrapper_attributes();
 	};
 	add_filter( 'next_comments_link_attributes', $filter_link_attributes );
```

Comment:
```suggestion
	add_filter( 'next_comments_link_attributes', 'get_block_wrapper_attributes' );
```

Why not just do this?

Reply:
I think I know why this is done. `next_comments_link_attributes` the value is a string and `get_block_wrapper_attributes` expects an array.

Reply:
Yes, I see that `get_block_wrapper_attributes()` takes an `$extra_attributes` array arg.